.gitignore

```diff
@@ -34,3 +34,4 @@
 # unless supporting rvm < 1.11.0 or doing something fancy, ignore this:
 .rvmrc
 node_modules/
+*.log
```

Comment:
@noor-ahmed you can copy all from this https://github.com/nimblehq/git-template/blob/master/.gitignore

Reply:
@Lahphim Yeah i already did that, this I added to skip all `logs`